src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -48,4 +55,16 @@ export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
   );
 };
 
+// TODO: this was how CampAttendance looked before we deleted it
+// const CampAttendance: FC<PropsType> = ({ attendees }) => {
```

Comment:
This can be deleted

Reply:
removed in https://github.com/sparkletown/sparkle/pull/840/commits/881676de95fe02cb743898b3bff51818292514fa